### PR TITLE
id_card 21.jpg done

### PR DESCRIPTION
**Pull request by Bhosle07**

### CODE REVIEW

File: I card/Bhosle_Mahesh_21.jpg/id_card/index.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="top-right-small2"></div>
+            <div class="bottom-left2"></div>
+            <div class="bottom-right2"></div>
+            <div class="mid-bottom1"></div>
+            <div class="mid-bottom2"></div>
+        </div>
+
+        <!-- Content of ID Card Back Side -->
+        <div class="id-card-content">
+            <div class="id-number2">id no: 123456</div>
```
Comment (jaiminkumar1810) [inline I card/Bhosle_Mahesh_21.jpg/id_card/index.html:66]: There are no id-number2 class in sample change it according to sample code

File: I card/Bhosle_Mahesh_21.jpg/id_card/index.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="id-emergency-contact">
+                <div class="Email"><i class="fa-solid fa-envelope"></i>mbhosle061@gmail.com</div>
+                <div class="phone"><i class="fa-solid fa-phone"></i> +91 9665650029</div>
+            </div>
+
+            <!-- Blood Group -->
+            <div class="id-blood-group">
+                Blood Group:O+
+            </div>
+                <div class="dates">
```
Comment (jaiminkumar1810) [inline I card/Bhosle_Mahesh_21.jpg/id_card/index.html:85]: Remove this div class name dates, if you have any issue to set content of ID Back Card write separate CSS for each class eg. id-blood-group/id-dob/id-issue-date/id-expiry-date

File: I card/Bhosle_Mahesh_21.jpg/id_card/index.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            <!-- ID Number -->
+            <div class="id-number">Id no : 123456</div>
+            
+            <!-- Photo -->
+            <div class="user-photo">
+                <img src="img/profile.jpg" alt="Photo">
+            </div>
+
+            <!-- Name -->
+             <div class="my-name"><span>BHOSLE</span><H1>MAHI</H1></div>
```
Comment (jaiminkumar1810) [inline I card/Bhosle_Mahesh_21.jpg/id_card/index.html:37]: The class name should be **user-name** instead of my-name

**Review by jaiminkumar1810 — CHANGES_REQUESTED** [review]
Check the comment and update your code ASAP

**Review by Adarshovhal — COMMENTED** [review]
All Requirement are Done like, In css file mm and pt unit are used.
All classes in the previous.html file.
Background classes in the index.html file.
Code was good in structure.

**Pull request merged**